.claude/skills/creating-slide-decks/skeleton.html

```
@@ -0,0 +1,180 @@
+<pre><span class="p">$</span> &lt;the real command&gt;
+<span class="c"># &lt;what it shows&gt;</span></pre>
+    </div>
+
+    <div class="note"><span class="t">&lt;key point&gt;</span>&lt;the honest, accurate detail — no fluff&gt;</div>
+  </div>
+</section>
+
+<!-- Add more <section class="slide" data-title="..."> blocks. The script counts them automatically. -->
+
+<div class="deck-bottom">
+  <button class="navbtn" id="prev" aria-label="Previous slide">‹</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)